README.md

```diff
@@ -259,7 +259,65 @@ BundleKeys.endpoints := Map("akka-remote" -> Endpoint("tcp"))
 
 In the above, no declaration of `services` is required as akka remoting is an internal, cluster-wide TCP service.
 
-## play[23|24|25]-conductr-bundle-lib
+## play25-conductr-bundle-lib
```

Comment:
Tabs would be great to keep code snippets for different flavors at one place. However, also the documentation text is different between the flavors, e.g.: https://github.com/typesafehub/conductr-lib/pull/77/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8R270

Therefore I think the best user flow can only be achieved by having a section for every possible angle, e.g. Play 2.5 conductr-client-lib Java, Play 2.5 conductr-client-lib Scala. And yes, we also should apply that to the other sections.

Is it possible to create reusable text blocks in markdown, similar as templates in Twirl?

I've created this issue to improve the documentation structure: https://github.com/typesafehub/conductr-lib/issues/79